imports/plugins/core/ui-navbar/client/components/navbar/navbar.js

```diff
@@ -1,9 +1,82 @@
+import { Meteor } from "meteor/meteor";
 import { FlatButton } from "/imports/plugins/core/ui/client/components";
 import { Reaction } from "/client/api";
 import { Tags } from "/lib/collections";
+import { Template } from "meteor/templating";
 
+
+Template.notificationItem.onCreated(function () {
+  this.notification = ReactiveVar();
+
+  // Create an auto run to Check for notifications on page load
+  // and set the notification reactive variable.
+  this.autorun(() => {
+    const instance = this;
+    Meteor.call("notifications/getNotifications", Meteor.userId(), (err, res) => {
+      instance.notification.set(res);
+    });
+  });
+});
+
+Template.notificationDropdown.onCreated(function () {
+  this.notification = ReactiveVar();
+
+  // Create an auto run to Check for notifications on page load
+  // and set the notification reactive variable.
+  this.autorun(() => {
+    const instance = this;
+    Meteor.call("notifications/getNotifications", Meteor.userId(), (err, res) => {
+      instance.notification.set(res.length);
+    });
+  });
+});
+
+Template.dropDownNotifications.events({
+    /**
+   * Clear Notifications
+   * @param  {Event} event - jQuery Event
+   * @return {void}
+   */
+  "click #clearNotifications": (event) => {
+    event.preventDefault();
+    Meteor.call("notifications/clearNotifications", Meteor.userId(), (err, res) => {
+      // Do nothing
+    });
+  }
+});
+
+Template.notificationDropdown.helpers({
+  NotificationIcon() {
+  // Check if the user has pending notifications
+  // and set the appropriate Icon
+    return (Template.instance().notification.get() > 0)
+    ? "fa fa-bell"
+    : "fa fa-bell-o";
+  },
+  checkNotification() {
+    return (Template.instance().notification.get() > 0)
+    ? true
+    : false;
+  }
+});
+Template.notificationItem.helpers({
+  showNotification() {
+    // Change the display state of the notification to show the latest notification when clicked
+    return Template.instance().notification.get();
+  }
+});
 Template.CoreNavigationBar.onCreated(function () {
   this.state = new ReactiveDict();
+  this.notification = ReactiveVar();
```

Comment:
Thanks for pointing this out, I have made the changes to other instances where it is an array, however, this particular case is not an array.